test/image/mocks/annotations.json

```diff
@@ -17,6 +17,7 @@
             {"text":"right bottom","showarrow":false,"xref":"paper","yref":"paper","xanchor":"right","yanchor":"bottom","x":0.5,"y":1},
             {"text":"move with page","xref":"paper","yref":"paper","x":0.75,"y":1},
             {"text":"opacity","opacity":0.5,"x":5,"y":5},
+            {"text":"not-visible", "visible": false},
```

Comment:
@alexcjohnson good :eye: 

I chose to revert a2dd634 and test annotations and shapes autorange using jasmine tests in 5e62c08053c4895f57bfd7443aa1894fb1883f08